specs/binding-http.spec/src/test/java/io/aklivity/zilla/specs/binding/http/internal/HttpFunctionsTest.java

```diff
@@ -320,15 +320,15 @@ public void shouldFailWhenBufferDoNotHaveEnoughSpace() throws Exception
     @Test
     public void shouldMatchDataExtension() throws Exception
     {
-        BytesMatcher matcher = HttpFunctions.matchDataEx()
+        BytesMatcher matcher = HttpFunctions.matchFlushEx()
```

Comment:
Need to rename method to `shouldMatchFlushExtension`, similar renames needed for other `...DataEx...` methods.